src/holofuel/pages/Inbox/Inbox.js

```diff
@@ -85,15 +110,19 @@ export default function Inbox () {
   const { actionableTransactions, recentTransactions } = useTransactionsWithCounterparties()
   const payTransaction = useOffer()
   const declineTransaction = useDecline()
-  const [toggleModal, setToggleModal] = useState(null)
+  const refundTransaction = useRefund()
+  const [counterpartyNotFound, setCounterpartyNotFound] = useState(true)
+  const [isNewTransactionModalVisible, setIsNewTransactionModalVisible] = useState(null)
   const [modalTransaction, setModalTransaction] = useState(null)
 
   const showConfirmationModal = (transaction = {}, action = '') => {
     const modalTransaction = { ...transaction, action }
     if (!isEmpty(transaction) && action !== '') return setModalTransaction(modalTransaction)
-    else return setToggleModal(true)
+    return null
```

Comment:
We don't care about the return value of `showConfirmationModal`, we only call it for it's side effect, so we should not return anything from it. 
Line 120 should be 
```
if (!isEmpty(transaction) && action !== '') setModalTransaction(modalTransaction)
```
and you can remove the `return null` line altogether